charts/umbrella/values.yaml

```
@@ -207,6 +207,91 @@ portal:
       senderEmail: "smtp@tx.test"
       password: ""
     portalmigrations:
+      seeding:
+        testData:
+          configMap: "portal-testdata"
```

would be great if you could add just a validation to the configMap instead:
`{{- if .Values.portal.backend.portalmigrations.seeding.testData.enabled-}}`
and:
```suggestion
        testData:
          enabled: true
          configMap: "portal-testdata"
```